library/src/lib/icon/icon.directive.ts

```diff
@@ -30,6 +45,7 @@ export class IconDirective extends AbstractFdNgxClass {
         }
     }
 
+    /** @hidden */
     constructor(@Inject(ElementRef) elementRef: ElementRef) {
```

Comment:
can you remove the @Inject(ElementRef) and replace with just 'private' as part of this pr pls?

Reply:
done